cases/management/commands/enrich_ciaa_allegations.py

```
@@ -267,379 +650,566 @@ def _get_ciaa_cases(
+                    temperature=0.1,
+                    timeout=timeout,
                 )
-                if combined:
-                    clean.append(combined)
-
-        if not clean:
-            return None
+                raw = response.content[0].text
+                logger.debug(f"LLM response: {raw[:100]}...")
+                return self._parse_allegations(raw)
+            except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)